releng/sap-kyma-runtime/pom.xml

```diff
@@ -0,0 +1,118 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>com.sap.xsk</groupId>
+		<artifactId>xsk-releng-parent</artifactId>
+		<version>0.16.0-SNAPSHOT</version>
+		<relativePath>../pom.xml</relativePath>
+	</parent>
+
+	<name>XSK - Releng - Kyma</name>
+	<artifactId>xsk-releng-kyma-runtime</artifactId>
+	<version>0.16.0-SNAPSHOT</version>
+	<packaging>war</packaging>
+
+	<build>
+		<finalName>${project.artifactId}</finalName>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-war-plugin</artifactId>
+				<configuration>
+					<failOnMissingWebXml>true</failOnMissingWebXml>
+					<webXml>src/main/webapp/WEB-INF/web.xml</webXml>
+					<warName>ROOT</warName>
+				</configuration>
+			</plugin>
+
+			<plugin>
+				<artifactId>maven-resources-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>copy-dynamic-data-source</id>
+						<phase>install</phase>
+						<goals>
+							<goal>copy-resources</goal>
+						</goals>
+						<configuration>
+							<outputDirectory>target</outputDirectory>
+							<overwrite>true</overwrite>
+							<resources>
+								<resource>
+									<directory>../../modules/basis/basis-datasource/target</directory>
+									<includes>
+										<include>*.jar</include>
+									</includes>
+								</resource>
+							</resources>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>			
+		</plugins>
+		<resources>
+	        <resource>
+	          <directory>src/main/resources</directory>
+	          <filtering>true</filtering>
+	        </resource>
+	    </resources>
+	</build>
+
+	<dependencies>
+		<!-- XS Classic -->
+		<dependency>
+			<groupId>${project.groupId}</groupId>
+			<artifactId>xsk-modules-all</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>${project.groupId}</groupId>
+			<artifactId>xsk-resources-all</artifactId>
+			<version>${project.version}</version>
+		</dependency>
```

Comment:
as discussed offline we should exclude IDE modules and Neo-sdk modules

Reply:
Done